code/modules/admin/topic.dm

```diff
@@ -1099,9 +1099,16 @@
 			to_chat(usr, "This can only be used on instances of type /mob/living/carbon/human.", confidential = TRUE)
 			return
 
+		var/move = TRUE
+		switch(alert(usr,"Move new AI to AI spawn location?","Move AI?", "Yes", "No","Cancel"))
+			if("Cancel")
+				return
+			if("No")
+				move = FALSE
+
 		message_admins("<span class='danger'>Admin [key_name_admin(usr)] AIized [key_name_admin(H)]!</span>")
```

Comment:
You need to check `!QDELETED(H)` after the prompt.

Reply:
Done, now throws a message and early returns if H was qdel'd.